README.md

```diff
@@ -23,30 +23,72 @@ To install a dependency in a specific package, specify the workspace:
 npm i lodash -w @coveo/headless-react-samples
 ```
 
+## Build
+
 To build all projects for production, run:
 
 ```sh
 npm run build
 ```
 
-To build a single project for production (for instance, the `@coveo/atomic` package), run:
+To build a single project for production (for instance, the `atomic` package), run:
+
+```sh
+npx nx run atomic:build
+```
+
+## Development mode
+
+To start Atomic & Headless simultaneously in development (recommended), run:
 
 ```sh
-npm run build -w @coveo/atomic
+npx nx run atomic:dev
 ```
 
 To start a single project in development (for instance, the `quantic` package), run:
 
 ```sh
-npm dev -w @coveo/quantic
+npx nx run quantic:dev
 ```
 
-To start Atomic & Headless simultaneously in development (recommended), run:
+To run a specific task in a package separate it with colon e.g. to run `test:watch` inside quantic
+```sh
+npx nx run quantic:test:watch
+```
 
+To start story book in development, run:
 ```sh
-npm run dev:atomic
+npx nx run atomic-storybook:dev
 ```
 
+## Test
+
+To run the tests for a specific package (recommended) e.g. `atomic` package
+```sh
+npx nx run atomic:test
+```
+
+For e2e tests
+```sh
+npx nx run atomic:dev
+# In a separate terminal
+npx nx run atomic:e2e
+```
+
+To run e2e tests for specific files/components using the Cypress GUI
+```sh
+npx nx run atomic:e2e:watch
```

Comment:
There is one unfortunate edge-case with Nx: it splits on every `:` character, and assumes that the third item is [an executor configuration](https://nx.dev/recipes/executors/use-executor-configurations).

You can make Nx ignore the second colon symbol by using quotation marks like so:
```suggestion
npx nx run atomic:"e2e:watch"
```

Reply:
works as expected without quotes e.g. it opens cypress in watch mode for following
```
ui-kit % npx nx run atomic:e2e:watch
...
>  NX   Successfully ran target e2e:watch for project atomic
```